packages/dds/tree/src/core/tree/anchorSet.ts

```diff
@@ -982,7 +1004,8 @@ export class AnchorSet implements Listenable<AnchorSetRootEvents>, AnchorLocator
 				this.notifyChildrenChanging();
 				this.detachEdit(range, oldContentDestination);
 				this.attachEdit(newContentSource, range.end - range.start, range.start);
-				this.notifyChildrenChanged();
+				assert(this.parentField !== undefined, "Must be in a field in order to attach");
```

Comment:
Not sure if this was intentional. I can't say that I care very much.
```suggestion
				assert(this.parentField !== undefined, "Must be in a field in order to replace");
```

Reply:
Copy paste; I'll fix them. Might also move the assert to inside notifyChildrenChanged.